### ISSUE TITLE
Nothing about vox blood transfusion makes sense

### ISSUE THREAD

**Issue by Exxion** [d3athrow/vgstation13#3295]
It really seems like vox and human blood shouldn't be interchangeable. Really, considering the circumstances, human blood should be highly toxic to vox and vice versa.
Human blood contains a lot of oxygen. Carrying it is sort of its main purpose. Not only is oxygen more toxic to vox than plasma is, but human blood doesn't even carry the nitrogen that vox breathe (except in useless dissolved form).
Given its color and the fact that vox breathe nitrogen, vox blood is probably based on some cyanide-containing compound. There are a lot of cyanide compounds with a lot of different effects on the human body, but vox blood should at best be useless to humans and at worst massively toxic. (This also means that vox should probably take carbon or something rather than iron to increase blood production, but that's a different issue.)
Also, giving a vox human blood doesn't change the blood color of the vox.

I propose adding vox bloodbags. They would work exactly like human bloodbags except be blue and used by and for vox. Start with a few in medbay and piles in the trading post.

Also, making the blood varieties toxic to the species they are not from could lead to some pretty interesting ways to poison targets. For balance reasons, they probably shouldn't be as toxic as logic/chemistry would suggest, but somewhat toxic seems fine.

TL;DR: IMERSION BROKED

**Comment by 9600bauds** [comment]
Also, Vox shouldn't be able to give/receive CPR from humans. But Medbay is already pretty ill-suited for aliens, their cloners couldn't even make exotic anatomy until about a month ago. Maybe NT just doesn't give a shit.

**Comment by Probe1** [comment]
>Giving Vox CPR poisons them

CODE IT NOW YES YES YES

**Comment by ComicIronic** [comment]
> Vox shouldn't be able to give/receive CPR from/to humans.

Not to get overly technical, but the main goal of CPR is to keep airflow going through the lungs and keep the heart beating. While a Vox receiving CPR would technically suffer slight toxins from oxygen, it would be more help than hurt to keep their blood pumping and a nitrogen exchange going.

**Comment by 9600bauds** [comment]
Once again Comic, you're not wrong there.

**Comment by Exxion** [comment]
I'd expect a vox receiving CPR to get more than slight toxins, but it definitely would also keep them from suffocating, given that air is still 80% nitrogen.
1% oxygen content is enough to give vox moderate toxins damage over a relatively short time. Regular air will deal a good bit of damage just in the time it takes to notice internals are off and turn them on.
Then again, toxin damage is probably the least of your worries if you're in crit. There's no limit on how much antitox you can take anyway.

I wonder if a vox with an antitox IV could just breathe air

**Comment by 9600bauds** [comment]
They can, it's the only way to have a smoke in this lousy-ass station.

**Comment by Beroya** [comment]
With modern-day CPR we don't even breathe into the patient's lungs anymore, it's just pumping the heart.
